data/data_utils.py

```diff
@@ -60,8 +60,10 @@ def load_device(self, device, truncate=False, max_days=None,
 
         # Handle the validation data, which can be truncated but uses the same
         # Data directory as the train set. 
+        was_valid = False
```

Comment:
Need a flag to store whether it was a validation dataset before over writing the validation set to reference the train set.